static/app/views/issueDetails/utils.tsx

```diff
@@ -167,3 +168,32 @@ export function useEnvironmentsFromUrl(): string[] {
 
   return envsArray;
 }
+
+export function getGroupDetailsQueryData({
+  environments,
+}: {
+  environments?: string[];
+} = {}): Record<string, string | string[]> {
+  // Note, we do not want to include the environment key at all if there are no environments
+  const query: Record<string, string | string[]> = {
+    ...(!isEmpty(environments) ? {environment: environments} : {}),
+    expand: ['inbox', 'owners'],
```

Comment:
i think we can drop owners, no longer being used

Reply:
Nice, good catch. Will follow up with that